content/md/what-is/what-is-a-rule.md

```diff
@@ -55,9 +55,10 @@ rules:
             -   type: set
                 field: camera_brand
                 value: canon_brand
-``` 
+```
```

Comment:
on line 50, I'm not sure we have to specify `.code` anymore, I guess 
```yaml
- field: camera_brand
  operator: 'NOT IN'
  value:
    - canon_brand
```
is enough